src/Get.tsx

```diff
@@ -143,12 +154,12 @@ class ContextlessGet<T> extends React.Component<GetComponentProps<T>, Readonly<G
     };
   };
 
-  fetch = async (requestPath?: string, thisRequestOptions?: RequestInit) => {
+  public fetch = async (requestPath?: string, thisRequestOptions?: RequestInit) => {
     const { base, path } = this.props;
     this.setState(() => ({ error: "", loading: true }));
 
     const { resolve } = this.props;
-    const foolProofResolve = resolve || (data => data);
+    const foolProofResolve = resolve || (noop => noop);
```

Comment:
Technically, `noop` means no operation, and it's a classic function name that represent `() => {}`.

I think that I prefer the old `data => data` here ;)